src/cmdline/mod.rs

```diff
@@ -281,6 +281,16 @@ pub struct CommonCustomizeConfig {
     /// including when Ignition is run.
     #[clap(long, value_name = "path")]
     pub network_keyfile: Vec<String>,
+    /// Nmstate file to generate NetworkManager keyfile for live & dest
+    ///
+    /// Configure networking using the generated NetworkManager keyfile.
```

Comment:
Maybe let's say something like "using a NetworkManager keyfile generated from the specified Nmstate config"?

Reply:
Done